From a7631c9d203c0a318e2064e0234798d2b157d274 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=EC=B5=9C=EC=98=88=EC=A7=84?= <cyjbbh@gmail.com>
Date: Mon, 31 Mar 2025 01:31:01 +0900
Subject: [PATCH] =?UTF-8?q?ip=20=EC=A3=BC=EC=86=8C=20=EC=82=AD=EC=A0=9C?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
 ip주소.txt | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 ip주소.txt

diff --git a/ip주소.txt b/ip주소.txt
new file mode 100644
index 0000000..01442d9
--- /dev/null
+++ b/ip주소.txt
@@ -0,0 +1 @@
+ip 주소: 192.168.211.28
\ No newline at end of file
-- 
2.46.0.windows.1

